lib/authentication.js

```diff
@@ -118,7 +120,7 @@ if (config.github.requireTeam) {
       });
```

Comment:
The `getTeamMembership()` call doesn't appear in the docs. I'll have to update this a bit more.

Reply:
Fixed.